data/foss2cyber.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "Ishan Velle",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Front-end Web Developer | Freelancer | Designer | Blogger"
+  "avatar": "https://github.com/foss2cyber.png"
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/thefoss_m",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Here's my Github profile",
+      "url": "https://github.com/foss2cyber",
+      "icon": "FaGithubAlt"
+    },
```

Remove this extra comma
```suggestion
    }
```

---

Removed it. :+1: